tools/testsys/src/run.rs

```diff
@@ -146,7 +146,7 @@ impl Run {
             .get(
                 &self
                     .repo
-                    .unwrap_or(test.repo.unwrap_or("default".to_string())),
+                    .unwrap_or_else(|| test.repo.unwrap_or_else(|| "default".to_string())),
```

Comment:
If this are cleaning changes, I'd prefer them to be in the commit that introduce the change (i. e. `target_cluster_names.len() == 0` was introduced in a different commit)